diff --git a/libqtile/backend/x11/xcbq.py b/libqtile/backend/x11/xcbq.py
index 6e5ceb5..e4581e9 100644
--- a/libqtile/backend/x11/xcbq.py
+++ b/libqtile/backend/x11/xcbq.py
@@ -523,17 +523,7 @@ class Connection:
     @property
     def pseudoscreens(self):
         pseudoscreens = []
-        if hasattr(self, "xinerama"):
-            for i, s in enumerate(self.xinerama.query_screens()):
-                scr = PseudoScreen(
-                    self,
-                    s.x_org,
-                    s.y_org,
-                    s.width,
-                    s.height,
-                )
-                pseudoscreens.append(scr)
-        elif hasattr(self, "randr"):
+        if hasattr(self, "randr"):
             for i in self.randr.query_crtcs(self.screens[0].root.wid):
                 scr = PseudoScreen(
                     self,
@@ -542,6 +532,17 @@ class Connection:
                     i["width"],
                     i["height"],
                 )
+                if i["width"] != 0 and i["height"] != 0:
+                    pseudoscreens.append(scr)
+        elif hasattr(self, "xinerama"):
+            for i, s in enumerate(self.xinerama.query_screens()):
+                scr = PseudoScreen(
+                    self,
+                    s.x_org,
+                    s.y_org,
+                    s.width,
+                    s.height,
+                )
                 pseudoscreens.append(scr)
         return pseudoscreens
 
