--- hardware-configuration.nix	2022-11-01 22:26:56.127280502 +0900
+++ _hardware-configuration.nix	2022-11-01 22:28:40.199740501 +0900
@@ -9,18 +9,18 @@
     ];
 
   boot.initrd.availableKernelModules = [ "ahci" "nvme" "xhci_pci" "virtio_pci" "usbhid" "usb_storage" "uas" "sr_mod" "sd_mod" "virtio_blk" ];
-  boot.initrd.kernelModules = [ "dm-snapshot" ];
+  # boot.initrd.kernelModules = [ "dm-snapshot" ];
   boot.kernelModules = [ "kvm-amd" "kvm-intel" ];
   # boot.resumeDevice = "/.swapfile";
   # boot.kernelParams = [ "resume_offset=resume_size" ];
   boot.extraModulePackages = [ ];
-  boot.initrd.luks.devices = {
-    luksroot = {
-      device = "/dev/disk/by-partlabel/GENERALLUKSROOT";
-      preLVM = true;
-      allowDiscards = true;
-    };
-  };
+  # boot.initrd.luks.devices = {
+  #   luksroot = {
+  #     device = "/dev/disk/by-partlabel/GENERALLUKSROOT";
+  #     preLVM = true;
+  #     allowDiscards = true;
+  #   };
+  # };
 
   fileSystems."/" =
     { 
